### PR TITLE
Fix AJAX counter, improve test logging, share integration test setup

### PR DESCRIPTION
**Pull request by seanf**

### CODE REVIEW

**Comment by seanf** [comment]
@djansen-redhat Could you please look through this too?

**Comment by carlosmunoz** [comment]
:+1: Looks good from my side. The maven changes are a bit of a nightmare to follow, but I guess we can polish in the future if there's room for it.

**Comment by seanf** [comment]
Thanks @carlosmunoz .  

@huangp Have you got some time to take a look too?

File: zanata-war/src/test/resources/arquillian.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <container qualifier="jbossas-managed">
     <configuration>
       <!-- We can use the JBOSS_HOME env variable instead of this -->
-      <property name="jbossHome">${arquillian.jboss.home}</property>
+      <property name="jbossHome">${jboss.home}</property>
```
Comment (huangp) [inline zanata-war/src/test/resources/arquillian.xml]: I remember @carlosmunoz is against this idea but can't remember exactly why. Carlos can you explain again?

**Comment by huangp** [comment]
:+1: except arquillian jboss home question

File: zanata-war/src/test/resources/arquillian.xml (hunk 1 above)
Comment (carlosmunoz) [inline zanata-war/src/test/resources/arquillian.xml]: I honestly can't remember that discussion. In any case, I'm good with it now :smile:

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested (mostly via Wildfly8 - Jenkins handled the rest just fine)

**Pull request merged**